### ISSUE TITLE
[BUG-A11Y-007] TC-A11Y-007: Interactive Icons Fail Colour Contrast (WCAG AA) Standard Requirements

### ISSUE THREAD

**Issue by PurpleDotte** [PLP-Database-Design/wk-6-1-PurpleDotte#7]
**Steps to reproduce:**
1. Render BookStore web app
2. Click on “WAVE” icon in your browser toolbar
3. In the WAVE sidebar, select the “Contrast“ tab
4. For any failures listed in the Contrast tab, expand the item and look at the details or click on the failure item on the catalog page.

 **Expected:**
- Text contrast ratio ≥ 4.5:1 for normal text
- No WCAG AA contrast failures
- Interactive elements meet contrast requirements

**Actual:**
- Text contrast ration < 4.5:1 for normal text (3.76:1)
- WCAG AA contrast fails
- Interactive elements don’t meet contrast requirements

**Environment:**
Browser:
Chrome: version 142.0.7444.135 (Official Build) (64-bit).
Firefox Browser: 144.0.2 (64-bit)
Microsoft Edge: version 142.0.3595.69 (official build) (64-bit)

OS: Windows 11
Device: Desktop

**Severity:** Medium
**Priority:** High
**Affected FR:** FR-X01

**Evidence:**

Chrome
<img width="1362" height="717" alt="TC-A11Y-007: Color Contrast - WCAG AA" src="https://github.com/user-attachments/assets/85d9dbf7-eb41-4df1-906e-7a0aead9cec1" />

Edge
<img width="1361" height="726" alt="TC-A11Y-007: Color Contrast - WCAG AA" src="https://github.com/user-attachments/assets/ad6eed82-29d6-4b05-9492-cc12d72b2bbb" />

Firefox
<img width="1313" height="715" alt="TC-A11Y-007: Color Contrast - WCAG AA" src="https://github.com/user-attachments/assets/835cc7c3-723f-41e6-87c4-18087d2b77dc" />
